rainloop/l10n/fr.js

```diff
@@ -0,0 +1,12 @@
+OC.L10N.register(
+    "rainloop",
+    {
+    "Email" : " E-mail",
+    "RainLoop Webmail" : "RainLoop Webmail",
+    "Go to RainLoop Webmail admin panel" : "Se rendre sur la page d'administration de RainLoop Webmail",
+    "Let the user define its own email/password credentials only" : "Uniquement laisser l'utilisateur définir ses propres identifiants email/mot de passe",
```

Comment:
Contre-proposition: "Laisser à l'utilisateur le choix de se connecter automatiquement avec des identifiants saisis manuellement par lui"
What do you think? Clearer?

Reply:
Yes better.
I'd change the end as "Laisser à l'utilisateur le choix de se connecter automatiquement avec des identifiants qu'il saisira manuellement"

Reply:
all good!